src/backend/marsha/core/tests/test_api_video_share_live_media.py

```diff
@@ -0,0 +1,1190 @@
+                    "timed_text_tracks": [],
+                    "urls": {
+                        "manifests": {
+                            "hls": "https://channel_endpoint1/live.m3u8",
+                        },
+                        "mp4": {},
+                        "thumbnails": {},
+                    },
+                    "should_use_subtitle_as_transcript": False,
+                    "has_transcript": False,
+                    "playlist": {
+                        "id": str(shared_live_media.video.playlist.id),
+                        "title": "foo bar",
+                        "lti_id": "course-v1:ufr+mathematics+00001",
+                    },
+                    "shared_live_medias": [
+                        {
+                            "active_stamp": None,
+                            "filename": None,
+                            "id": str(shared_live_media.id),
+                            "is_ready_to_show": False,
+                            "nb_pages": shared_live_media.nb_pages,
+                            "show_download": True,
+                            "title": None,
+                            "upload_state": "pending",
+                            "urls": None,
+                            "video": str(video.id),
+                        }
+                    ],
+                    "live_state": "running",
+                    "live_info": {
+                        "jitsi": {
+                            "config_overwrite": {},
+                            "domain": "meet.jit.si",
+                            "external_api_url": "https://meet.jit.si/external_api.js",
+                            "interface_config_overwrite": {},
+                        },
+                        "medialive": {
+                            "input": {
+                                "endpoints": [
+                                    "https://live_endpoint1",
+                                    "https://live_endpoint2",
+                                ],
+                            }
+                        },
+                    },
+                    "live_type": JITSI,
+                    "xmpp": {
+                        "bosh_url": "https://xmpp-server.com/http-bind?token=xmpp_jwt",
+                        "websocket_url": None,
+                        "conference_url": f"{video.id}@conference.xmpp-server.com",
+                        "jid": "conference.xmpp-server.com",
+                    },
+                },
+                content,
+            )
+            self.assertEqual(video.active_shared_live_media, shared_live_media)
+            self.assertEqual(video.active_shared_live_media_page, 2)
+            mock_broadcast_message.assert_called_with(
+                str(video.id), json.dumps(content)
+            )
+
+    def test_api_video_shared_live_media_navigate_no_shared(self):
+        """An instructor can not navigate in an unexisting shared live media."""
```

Comment:
This is not enough. A video can have multiple `shared_live_media` but none active. You are not testing this scenario and you code in the api does not handle this case.